docs/tactics.md

```diff
@@ -186,6 +186,23 @@ example {α : Type*} {a b : α} [add_comm_group α] (hyp : a + a - a = b - b) :
 by { abel at hyp, exact hyp }
 ```
 
+### norm_num
+
+Normalises numerical expressions. It supports the operations
```

Comment:
This is phrased kind of funny. `<` isn't an operation and I'm not sure what it means to "support" `<` in numeral normalization. Maybe it would be clearer to describe the normalization behavior separately from the proving behavior?

I can't check right this sec, but doesn't `norm_num` prove `nat.prime` too?

Reply:
I copied this stuff from the docstring ;-) I just assumed it was correct CS language!

Reply:
I've rewritten. Is it OK? The `norm_num` docstring is currently this:

```
/-- Normalize numerical expressions. Supports the operations
  `+` `-` `*` `/` `^` `<` `≤` over ordered fields (or other
  appropriate classes), as well as `-` `/` `%` over `ℤ` and `ℕ`. -/
```

Shall I change this too? I am not sure why we say it supports - over a field and also - over nat, we can just say it supports - over a wide variety of things.